app/models/concerns/audit_extensions.rb

```diff
@@ -33,9 +34,27 @@ def ensure_taxonomies_not_escalated
   private
 
   def log_audit
-    Foreman::Logging.with_fields(self.audited_changes) do
-      Foreman::Logging.logger('audit').info { "#{self.action} event for #{self.auditable_type} with id #{self.auditable_id}" }
+    telemetry_increment_counter(:audit_records_created, 1, type: self.auditable_type)
+    aulogger = Foreman::Logging.logger('audit')
```

Comment:
please rename `aulogger` to `audit_logger` or at least `au_logger` to be consistent, we use underscores in ruby for multi words names and usually we don't abbreviate